src/testing/euler_naca0012.cpp

```diff
@@ -206,7 +204,7 @@ ::run_test () const
             //std::shared_ptr<HighOrderGrid<dim,double>> joukowski_mesh = read_gmsh <dim, dim> ("joukowski_R1_Q"+std::to_string(grid_degree)+".msh");
             //std::shared_ptr<HighOrderGrid<dim,double>> naca0012_mesh = read_gmsh <dim, dim> ("new_msh41.msh");
             //std::shared_ptr<HighOrderGrid<dim,double>> naca0012_mesh = read_gmsh <dim, dim> ("naca0012_hopw.msh");
-            std::shared_ptr<HighOrderGrid<dim,double>> naca0012_mesh = read_gmsh <dim, dim> (mesh_filename);
+            std::shared_ptr<HighOrderGrid<dim,double>> naca0012_mesh = read_gmsh <dim, dim> ("naca0012.msh");
```

Comment:
Revert this change

Reply:
Different meshes can be used, like the test that is to be uncommented in issue #121

Reply:
Unless the plan is to move that test to FlowSolver

Reply:
It up to you if you want to move the test to FlowSolver and get rid of euler_naca0012.cpp. The only thing is that there won't be any more grid study or polynomial ramping, since FlowSolver isn't really set up for that.

Reply:
What I meant was, replace the section of code inside `euler_naca0012.cpp` that essentially executes what the flow solver `run_test()` function does. This way the setup and time advancement is all accomplished inside there. Is this possible?  Like in the for loop you create or update an `all_parameters` object, then do `create_FlowSolver(all_parameters)` then `flow_solver->run_test()` and use the updated members like `flow_solver->dg` to do the rest of the operations in the `euler_naca0012.cpp` test

Reply:
Check out my most recent commit.